.github/workflows/auto-merge.yml

```diff
@@ -0,0 +1,33 @@
+name: Dependabot auto-merge
+
+on:
+  pull_request_target:
+  push:
+  workflow_dispatch:
+
+permissions:
+  pull-requests: write
```

Comment:
Is this permission needed?

Reply:
The [docs](https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/automating-dependabot-with-github-actions#enable-auto-merge-on-a-pull-request) list this and the content permission.  From reading around the `pull_request_target` event I'm pretty sure this is actually the permission we'll need during testing but I've kept both as a just in case strategy.

Reply:
But we don't need this permission for this investigation, do we?

Reply:
I suspect this is only necessary for the `gh pr` commands in that docs link but wanted to keep everything as close to the docs as possible while investigating this.